### DISCUSSION TITLE
Live Share not available and working

### DISCUSSION THREAD

**Discussion by flameface** [coder/code-server#4810] [General]
Yea! I am here to ask about an extension live Share by microsoft where we can code live with other, and it's not there in code server, i have tried to install it for browser and install it but still it doesn't works after installing, can anyone go through it

**Comment by lg-nineoneninethreenine** [comment]
code-server doesn't use Microsoft's extension registry, it uses Open-VSX. You can install extensions by downloading it with `wget` and then running `code-server --install-extension extension-filename`. Replace extension-filename with your extension file name. I don't think Live Share exists on Open VSX. You could try those steps above, and as for getting the VSIX extension file, go to the extension in your browser, hit F3 and search for Download. Find Download VSIX, copy the link via right-click and then use `wget`.

**Reply by flameface** [reply]
> code-server doesn't use Microsoft's extension registry, it uses Open-VSX. You can install extensions by downloading it with `wget` and then running `code-server --install-extension extension-filename`. Replace extension-filename with your extension file name. I don't think Live Share exists on Open VSX. You could try those steps above, and as for getting the VSIX extension file, go to the extension in your browser, hit F3 and search for Download. Find Download VSIX, copy the link via right-click and then use `wget`.



> code-server doesn't use Microsoft's extension registry, it uses Open-VSX. You can install extensions by downloading it with `wget` and then running `code-server --install-extension extension-filename`. Replace extension-filename with your extension file name. I don't think Live Share exists on Open VSX. You could try those steps above, and as for getting the VSIX extension file, go to the extension in your browser, hit F3 and search for Download. Find Download VSIX, copy the link via right-click and then use `wget`.

I have tried already by downloading it and installing and it gets installed when i press start or join buttons it doesn't work

**Reply by lg-nineoneninethreenine** [reply]
So, it has to be a code-server bug or Live Share doesn't support it. How about Duckly? There's documentation on Coder.com that will help you get it on your instance.

**Reply by jsjoeio** [reply]
Yes! Try Duckly: https://coder.com/docs/code-server/latest/collaboration#code-sharing-using-duckly
